docs/index.md

```diff
@@ -16,7 +16,7 @@ PMM helps you improve the performance of your databases, simplify their manageme
 Here's how the home page looks on our <a href='https://pmmdemo.percona.com/' target='_blank'>free, live demo system</a>.
 
 <a href='https://pmmdemo.percona.com/' target='_blank'>
-    <img src="_images/PMM_Home_Dashboard.jpg" alt="PMM Demo Home Page" width=600px class="imgcenter"/>
+    <img src="_images/PMM_Home_Dashboard_1.png" alt="PMM Demo Home Page" width=800px class="imgcenter"/>
```

Comment:
I see you have updated image name here, but also updated the original image and added "updated" images. Why those changes are needed ?

Reply:
I was not getting the desired image quality. I was trying with a different resolution. So during the trial and error method, I did that.

Reply:
ok, so maybe you need to replace original image and don't add other one's ?